src/styles/_datavisyn.scss

```diff
@@ -28,24 +28,33 @@ $dataviyn_font: Monda !default;
 
 nav.navbar,
 nav.navbar-inverse {
+  min-height: auto; // Temp fix until https://github.com/phovea/phovea_clue/issues/140 is fixed
+
   .navbar-brand.caleydo_app {
     background: none;
     padding-left: 0;
+    display: flex;
+    align-items: center;
 
     &:before {
       @include datavisyn-font();
-      content: 'd';
-      font-size: 80%;
-      background: black;
-      border-radius: 5px;
-      padding: 0.2em 0.3em;
-      margin-left: 0.2em;
+      display: inline-block;
+      height:14px;
+      width:12px;
+      background: url('~tdp_core/src/assets/icons/datavisyn/datavisyn_newlogo.png') no-repeat 0 0;
```

Comment:
I think we can just name the file _datavisyn_logo.png_. And maybe we can use an SVG version?